services/QuillLMS/config/initializers/cookies_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# Be sure to restart your server when you modify this file.
+
+# Specify a serializer for the signed and encrypted cookie jars.
+# Valid options are :json, :marshal, and :hybrid.
+Rails.application.config.action_dispatch.cookies_serializer = :hybrid
```

Comment:
Nice that they provide a migration path. I don't think this is an issue for how we use cookies, but just flagging it in case it comes up:
> SPOILER! When using the :json or :hybrid serializer, you should be aware that not all [Ruby](https://www.botreetechnologies.com/blog/interesting-methods-to-work-with-ruby-hashes-in-a-better-way) objects can be serialized as JSON. For example, Date and Time objects will be serialized as strings, and Hashes will have their keys stringified.